src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -91,6 +89,11 @@ const useStyles = makeStyles(theme => ({
       left: '80%',
       bottom: theme.spacing(18),
     },
+    color: 'white',
+    textDecorationLine: 'none',
+    '&:hover': {
+      textDecorationLine: 'underline',
+    },
```

Comment:
This is cute, you put this in `icon` instead of `version` XD

Reply:
The rebase was going weird on me